src/desktop/components/react/auction_block/query.js

```diff
@@ -1,16 +0,0 @@
-export default `
-  cover_image {
-    cropped(width: 260, height: 110) {
-      url
-    }
-  }
-  end_at
-  href
-  id
-  is_closed
-  is_live_open
-  is_preview
-  live_start_at
-  name
-  start_at
-`
```

Comment:
I don't think we can remove this file yet since it's still being used in `src/desktop/apps/auction/queries/sale.js`:

https://github.com/artsy/force/blob/faa20a638791f6b6ad94a4cab54318bf68161997/src/desktop/apps/auction/queries/sale.js#L7